pkg/didcomm/dispatcher/api.go

```diff
@@ -32,6 +32,23 @@ type DIDCommMsg struct {
 	OutboundDestination *Destination
 }
 
+// MsgEventType msg event type
+type MsgEventType int
+
+const (
+	// PreState pre state
+	PreState MsgEventType = iota
+	// PostState post state
+	PostState
+)
+
+// MsgEvent msg event
+type MsgEvent struct {
```

Comment:
It’s a bit confusing that there are now 2 naming conventions for Register types.

Reply:
[Reference](https://github.com/hyperledger/aries-framework-go/pull/321#discussion_r327315477) to previous discussion on this topic

Reply:
@troyronda @fqutishat May be we can just name `Msg` instead of `MsgEvent` to fit the existing pattern. what do you think ?

Reply:
@troyronda @rolsonquadras I will change it to Msg